.hlint.yaml

```diff
@@ -0,0 +1,93 @@
+# HLint configuration file
+# https://github.com/ndmitchell/hlint
+##########################
+
+# This file contains a template configuration file, which is typically
+# placed as .hlint.yaml in the root of your project
+
+- arguments: [-XTypeApplications, -XRecursiveDo, --cpp-define=CONFIG=dev]
+
+- ignore: {name: Redundant do}
+- ignore: {name: Redundant bracket}
+- ignore: {name: Redundant $}
+- ignore: {name: Redundant flip}
+- ignore: {name: Move brackets to avoid $}
+- ignore: {name: Eta reduce}
+- ignore: {name: Avoid lambda}
+- ignore: {name: Use camelCase}
+- ignore: {name: Use const}
+- ignore: {name: Use if}
+- ignore: {name: Use notElem}
+- ignore: {name: Use fromMaybe}
+- ignore: {name: Use maybe}
+- ignore: {name: Use fmap}
+- ignore: {name: Use foldl}
+- ignore: {name: 'Use :'}
+- ignore: {name: Use ++}
+- ignore: {name: Use ||}
+- ignore: {name: Use &&}
+- ignore: {name: 'Use ?~'}
+- ignore: {name: Use <$>}
+- ignore: {name: Use .}
+- ignore: {name: Use head}
+- ignore: {name: Use String}
+- ignore: {name: Use Foldable.forM_}
+- ignore: {name: Unused LANGUAGE pragma}
+- ignore: {name: Use newtype instead of data}
+# Rules not found in old HLint.hs file (prior to HLint 2.0)
+# Added when we made the change.
+- ignore: {name: Redundant lambda}
+- ignore: {name: Use section}
+
+# Specify additional command line arguments
+#
+# - arguments: [--color, --cpp-simple, -XQuasiQuotes]
```

Comment:
It seems useless, because we are using this feature anyway.
In fact, I'd remove all these suggestions and leave a link to e. g. https://github.com/ndmitchell/hlint#customizing-the-hints, but that's up to you.